packages/types/src/electra/sszTypes.ts

```diff
@@ -14,8 +25,65 @@ import {ssz as bellatrixSsz} from "../bellatrix/index.js";
 import {ssz as capellaSsz} from "../capella/index.js";
 import {ssz as denebSsz} from "../deneb/index.js";
 
-const {UintNum64, Slot, Root, BLSSignature, UintBn256, Bytes32, BLSPubkey, DepositIndex, UintBn64, ExecutionAddress} =
-  primitiveSsz;
+const {
+  UintNum64,
+  Slot,
+  Root,
+  BLSSignature,
+  UintBn256,
+  Bytes32,
+  BLSPubkey,
+  DepositIndex,
+  UintBn64,
+  ExecutionAddress,
+  ValidatorIndex,
+} = primitiveSsz;
+
+export const AggregationBits = new BitListType(MAX_VALIDATORS_PER_COMMITTEE * MAX_COMMITTEES_PER_SLOT);
```

Comment:
Doesn't this type already exist from phase 0? Reuse if possible

Reply:
> Doesn't this type already exist from phase 0? Reuse if possible

The max length is different though. phase0 aggregation bits can only go up to `MAX_VALIDATORS_PER_COMMITTEE`, where as in electra it can go up to `MAX_VALIDATORS_PER_COMMITTEE * MAX_COMMITTEES_PER_SLOT `